x/genutil/gentx.go

```diff
@@ -113,6 +113,7 @@ func DeliverGenTxs(
 			panic(res.Log)
 		}
 	}
+	stakingKeeper.ExecuteEpoch(ctx)
```

Comment:
Could you add some docs for why we're executing an epoch here?

I think its because at genesis we want to know the new stake distribution immediately.

Reply:
We execute epoch here because we want to do delegations instantly before running node.